aerolyzer/test_aerosol.py

```diff
@@ -0,0 +1,61 @@
+'Aerosol.py tests'
+import time
+import numpy as np
+import aerosol
```

Comment:
@RossDan96 what is the aerosol library?Is this supposed to be part of the aerolyzer library which is then called here?